CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 ## Unreleased
 
+## v0.73.0
+
+This Splunk OpenTelemetry Collector release includes changes from the [opentelemetry-collector v0.73.0](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.73.0) and the [opentelemetry-collector-contrib v0.73.0](https://github.com/open-telemetry/opentelemetry-collector-contrib/releases/tag/v0.73.0) releases where appropriate.
+
+### 💡 Enhancements 💡
+- [Added profiling and JVM metrics options to linux installer script for zero configuration auto instrumentation of Java apps](https://github.com/signalfx/splunk-otel-collector/pull/2718)
+- [Update installer script with option to generate service name with zero configuration auto instrumentation of Java apps](https://github.com/signalfx/splunk-otel-collector/pull/2730)
```

Comment:
We can remove these 2 references to the installer script since they are included in the list below.

Reply:
done